map2loop/project.py

```diff
@@ -524,15 +526,35 @@ def save_into_projectfile(self):
         Creates or updates a loop project file with all the data extracted from the map2loop process
         """
         # Open project file
-        if self.loop_filename is None or self.loop_filename == "":
+        if not self.loop_filename:
             self.loop_filename = os.path.join(
                 self.map_data.tmp_path, os.path.basename(self.map_data.tmp_path) + ".loop3d"
             )
 
-        # Check overwrite of mismatch version
         file_exists = os.path.isfile(self.loop_filename)
+
+        if file_exists:
+            if self.overwrite_lpf:
+                try:
+                    os.remove(self.loop_filename)
+                    file_exists = False
+                    print(f"Existing file '{self.loop_filename}' was successfully deleted.")
+                except Exception as e:
+                    print(f"Failed to delete existing file '{self.loop_filename}': {e}")
+                    return
+            else:
```

Comment:
if it didn't work, we should probably just raise the exceptiuon. otherwise the stack trace is lost and it can be hard to find the cause.